tests/docker-compose.yml

```diff
@@ -9,6 +9,8 @@ services:
       - "${JAHIA_PORT}:8080"
     extra_hosts:
       - jahia:127.0.0.1
+    environment:
+      - TZ=America/Toronto
```

Comment:
@kovunov I don't think you should commit this in the compose or you should set-up TZ on all the servers in the compose, but here when running on CI, jahia wil on Toronto TZ and the other server will be on UTC